.github/workflows/docs-to-gh-pages.yml

```diff
@@ -14,9 +14,11 @@ jobs:
       - name: Setup Hugo
         uses: peaceiris/actions-hugo@v3
         with:
-          hugo-version: '0.110.0'
+          hugo-version: "0.110.0"
```

Comment:
According to [hugo docs](https://gohugo.io/hugo-pipes/transpile-sass-to-css/#installation-overview), dart sass is compatible with `0.114.0`.
Do we have to update the hugo-version here as well? 🤔

Reply:
Ahh I missed that, looks like that will need changing then. The action that just ran builds the old version so its not using dart-sass anyway